topics/community/tutorials/sig_create/tutorial.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: tutorial_hands_on
+priority: 2
+
+title: "Creating a Special Interest Group"
+questions:
+  - "What are the requirements for a Special Interest Group?"
+  - "How can I create a Special Interest Group?"
+  - "Where can I go to find Special Interest Group resources?"
+objectives:
+  - "Create a Special Interest Group"
+  - "Be aware of resources for expanding your Special Interest Group"
+time_estimation: "30m"
+key_points:
+  - "You can highlight questions, tools and hints with a special syntax"
+  - "Self-learning can be done by questions and hidden answers"
+requirements:
+-
+    type: "internal"
+    topic_name: community
+    tutorials:
+        - sig_define
+contributions:
+  authorship:
+  - nomadscientist
+  editing:
+  - shiltemann
+abbreviations:
+  SIG: Special Interest Group
+  CoP: Community of Practice
+  GCB: Galaxy Community Board
+  WG: Working Groups
+  GTB: Galaxy Technical Board
+  GEB: Galaxy Executive Board
+---
+
+So you have decided to create a new Special Interest Group (**SIG**)!
+
+> <warning-title>Do not contribute to software smog!</warning-title>
+> Make sure you have read our section [Do you need to make a SIG?]({% link topics/community/tutorials/sig_define/tutorial.md %}#do-you-need-to-make-a-sig) before creating a new **SIG**! Otherwise, you may be re-inventing an existing **SIG**!
+{: .warning}
+
+Let's walk you through making a new Galaxy **SIG**.
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Requirements of a Special Interest Group (SIG)
+
+Before starting a **SIG**, it's important to be aware of the (rather low) requirements for being a **SIG**:
+
+* Must have 2 or more members
+* Must maintain a page on the Community Hub with at least:
+  * A description of the purpose of the SIG.
+  * A clear explanation of how individuals can join and maintain membership.
+  * Details of access to the SIG’s communication channels.
+  * The current contact/email address for the nominated [Galaxy Community Board](https://galaxyproject.org/community/governance/gcb/) representative
+* Must manage membership and internal communication for itself.
+* Must use at least one public Matrix channel under the Galaxy Matrix space.
+* Must allow participation across different timezones (with the exception of **Regional Communities**).
+
+{% snippet topics/community/faqs/meeting_times.md %}
+
+{% snippet topics/community/faqs/governance_structure.md %}
+
+# Creating a new SIG
+
+Here are the steps for creating a SIG.
+
+1. Copy an updated SIG page (there's a Template in that folder, work in progress!) [from the Galaxy Community Hub github folder](https://github.com/galaxyproject/galaxy-hub/blob/master/content/community/sig/). We suggest looking through the different SIG pages to see which one has the type of content you would like and copying that one.
```

Comment:
This would be better as hands-on box here. Wdyt?

Reply:
NICE! Done